src/chain.js

```diff
@@ -61,10 +61,10 @@ class Chain {
  * @param {Chain} chain - Chain to compose the commit of.
  * @param {string} ecAddress - Entry Credit address that pays for the commit, either private (Es) or public (EC). 
  * If a public EC address is provided it is necessary to provide the signature of the commit as a 3rd argument (use case for hardware wallets)
- * @param {string|Buffer} [sig] - Optional signature of the commit (composeChainLedger). Only necessary if a public EC address was passed as 2nd argument.
+ * @param {string|Buffer} [signature] - Optional signature of the commit (composeChainLedger). Only necessary if a public EC address was passed as 2nd argument.
  * @returns {Buffer} - Chain commit.
  */
-function composeChainCommit(chain, ecAddress, sig) {
+function composeChainCommit(chain, ecAddress, signature) {
```

Comment:
I agree that now that this argument is publicly documented it is better to have a full name for the variable, but only renaming the argument is breaking the implementation inside the function. Actually inside the function I was also declaring a variable named `signature`, one possibility would be to rename the old `signature` to `sig` and vice versa.

Reply:
Good catch! How did I miss that. Thanks for reviewing.